### ISSUE TITLE
Problems with waking from Hibernate (Mettre en veille) and Suspend (Suspendre) functions of the User Indicator applet

### ISSUE THREAD

**Issue by PennyFarthing-GH** [getsolus/packages#4259]
### Please confirm there isn't an existing open bug report

- [X] I have searched open bugs for this issue

### Summary

The functions `Hibernate` (Mettre en veille) and `Suspend` (Suspendre) functions of the **User Indicator** applet are no longer operational (phenomenon observed since the kernel `linux-current` `6.10.11-303`, see attachment for details).

[Problems with Hibernate and Suspend functions of the User Indicator applet.txt](https://github.com/user-attachments/files/17638906/Problems.with.Hibernate.and.Suspend.functions.of.the.User.Indicator.applet.txt)

The problem persists with recent installation of the latest kernel (`6.11.5-307`).

### Steps to reproduce

2 scenarios after clicking user indicator button:
- select Hibernate
- select Suspend

### Expected result

- When `Hibernate`, then the computer must be off; when I put it on, I should get the display of the window of the application we were on before hibernating, and this, in the same place of its consultation.

- When `Suspend`, then the computer must be of (and that's the case); but I put it on, I should recover my session.

### Actual result

- When `Hibernate`, I got a black screen, the computer remaining on indefinitely; so I turned off my computer, then I turned it back on after 2 minutes -> the Gigabyte test pattern appeared, then the boot started and after 56 seconds, after logging in, the screen opened a new session.

- When `Suspend`, my computer turned off immediately; I turned it back on after 2 minutes -> there, no Gigabyte test pattern, a screen remaining black indefinitely; so I turned off my computer, then I turned it back on after 2 minutes -> the Gigabyte test pattern was displayed, then the boot started and after 53 seconds, after logging in, the screen opened a new session.

### Environment

- [X] Is system up to date?

### Repo

Shannon (stable)

### Desktop Environment

Budgie

### System details

```
administrateur@solus-pc ~ $ inxi -b
System:
  Host: solus-pc Kernel: 6.11.5-307.current arch: x86_64 bits: 64
  Desktop: Budgie v: 10.9.2 Distro: Solus 4.6 convergence
Machine:
  Type: Desktop Mobo: Gigabyte model: B85M-D3H v: x.x
    serial: <superuser required> UEFI: American Megatrends v: F15
    date: 08/20/2015
CPU:
  Info: dual core Intel Core i3-4160 [MT MCP] speed (MHz): avg: 3140
    min/max: 800/3600
Graphics:
  Device-1: Intel 4th Generation Core Processor Family Integrated Graphics
    driver: N/A
  Device-2: NVIDIA GP107 [GeForce GTX 1050 Ti] driver: nvidia v: 560.35.03
  Device-3: Logitech Webcam C270 driver: snd-usb-audio,uvcvideo type: USB
  Display: x11 server: X.Org v: 21.1.14 with: Xwayland v: 24.1.4 driver:
    gpu: nvidia,nvidia-nvswitch resolution: 1920x1080
  API: OpenGL v: 4.6.0 compat-v: 4.5 vendor: nvidia mesa v: 560.35.03
    renderer: NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2
Network:
  Device-1: Realtek RTL8111/8168/8211/8411 PCI Express Gigabit Ethernet
    driver: r8169
Drives:
  Local Storage: total: 1.36 TiB used: 821.98 GiB (58.8%)
Info:
  Memory: total: 16 GiB available: 15.5 GiB used: 2.33 GiB (15.0%)
  Processes: 236 Uptime: 43m Shell: Bash inxi: 3.3.36
administrateur@solus-pc ~ $ 
```

**Comment by TraceyC77** [comment]
*Possibly* addressed by [Display Driver 565.57.01](https://www.nvidia.com/en-us/drivers/details/233008/)

I've also had problems with wake from suspend with the current nvidia drivers (560.35.03 r548)
My laptop also has Intel / nvidia GPUs
When I try to wake from suspend, the laptop screen remains powered off. I hear system sounds. I can get a display by docking, which connects to an external screen. No matter what I do with the laptop display, it remains blank.

There is a [thread](https://forums.developer.nvidia.com/t/560-release-feedback-discussion/300830) on their forums for feedback on the 560 driver for those interested.

I don't know if this is something we can fix on our end, or if we are at the mercy of the upstream drivers.

**Comment by TraceyC77** [comment]
This is likely fixed with the latest nvidia driver. If you still have an issue, please contact nvidia support. Thanks.
